### PR TITLE
 Migrate ApplyNow Components to shadcn/Tailwind 

### PR DESCRIPTION
**Pull request by 0xgun**
DEPENDS on #14043

## Related Issue

#13946

### CODE REVIEW

File: src/components/Translatathon/ApplyNow.tsx
Comment (pettinarip) [inline src/components/Translatathon/ApplyNow.tsx:16]: Why would we want to change this?

File: src/components/Translatathon/ApplyNow.tsx
Comment (pettinarip) [inline src/components/Translatathon/ApplyNow.tsx:18]:
```suggestion
      <div className="pt-12">
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
@0xgun left one question. The rest looks good.

Seems that this PR was branched off #14043. Will mark it as blocked until that one gets merged.

File: src/components/Translatathon/ApplyNow.tsx
Comment (0xgun) [reply src/components/Translatathon/ApplyNow.tsx:16]: I forgot to change it back .

**Comment by 0xgun** [comment]
> @0xgun left one question. The rest looks good.
> 
> Seems that this PR was branched off #14043. Will mark it as blocked until that one gets merged.

@pettinarip pushed the changes

File: src/components/StablecoinsTable.tsx
Comment (pettinarip) [inline src/components/StablecoinsTable.tsx:46]:
```suggestion
    <Table className="my-8 min-w-[720px] bg-background">
```

File: src/components/Translatathon/ApplyNow.tsx
Comment (pettinarip) [inline src/components/Translatathon/ApplyNow.tsx:1]: We need to use the [new ButtonLink component](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/buttons/Button.tsx#L135) in this case.

**Review by pettinarip — CHANGES_REQUESTED** [review]
@0xgun left one last thing.

File: src/components/Translatathon/ApplyNow.tsx
Comment (0xgun) [reply src/components/Translatathon/ApplyNow.tsx]: made required changes

**Review by pettinarip — APPROVED** [review]

**Pull request merged**
